apps/src/StudioApp.js

```diff
@@ -1629,7 +1629,9 @@ StudioApp.prototype.runButtonClickWrapper = function (callback) {
  */
 StudioApp.prototype.configureDom = function (config) {
   var container = document.getElementById(config.containerId);
-  container.innerHTML = config.html;
+  if (typeof config.html !== 'undefined') {
+    container.innerHTML = config.html;
+  }
```

Comment:
When we do our top-level rendering with React (prior to StudioApp.init) we want to skip this step, which would clobber everything React just rendered.

In my prototype I originally skipped this with a more explicit config variable:
```javascript
if (!config.usingReactRendering) {
  container.innerHTML = config.html;
}
```
That felt a little like duplicating state though - why even assign config.html if we're not going to use it, and then why introduce another config variable to say we're not using config.html when we can just check if it's empty?

Alternatively, I could tear out this line altogether if I also give NetSim a top-level React wrapper (it's the one remaining user of this old approach).